.github/workflows/release.yaml

```diff
@@ -42,10 +42,73 @@ jobs:
       - name: Run tests
         run: tox -e unit
 
-  integration-test-lxd:
-    name: Integration tests (lxd)
+  charm-integration-test-lxd:
```

Comment:
You could replace these individual workflow calls with a simpler call to ci.yaml: 
```yaml
ci:
  uses: ./.github/workflows/ci.yaml
```